src/main/scala/consul/v1/session/SessionRequests.scala

```diff
@@ -30,7 +30,9 @@ object SessionRequests{
   private lazy implicit val SessionInfoReads     = Json.reads[SessionInfo]
   private lazy implicit val SessionDefWrites     = Json.writes[SessionDef]
 
-  def apply(basePath: String)(implicit executionContext: ExecutionContext): SessionRequests = new SessionRequests{
+  def apply(basePath: String)(implicit executionContext: ExecutionContext, rb: ConsulRequestBasics): SessionRequests = new SessionRequests{
+
+    import rb._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/Codacy/scala-consul/file/1255759094/issues/source?bid=2788492&fileBranchId=2939115#l35)